docs/guides/latest/auto-backup/database.md

```diff
@@ -11,3 +11,584 @@ product_name: stash
+
+### Install only YAMLs
+
+Run the following script to create only the YAMLs of the respective resources for `postgres-stash` catalog.
+
+```console
+curl -fsSL https://github.com/stashed/catalog/raw/master/deploy/script.sh | bash -s -- --catalog=postgres-stash
+```
+
+</div>
+<!-- ------------ Script Tab Ends----------- -->
+</div>
+
+Once installed, this will create `pg-backup-*` and `pg-recovery-*` Functions for all supported PostgreSQL versions. Verify that the Functions has been created successfully by.
+
+```console
+$ kubectl get function
+NAME            AGE
+pg-backup-9.6    6s
+pg-backup-10.2   6s
+pg-backup-10.6   6s
+pg-backup-11.1   6s
+pg-backup-11.2   6s
+pg-restore-9.6   6s
+pg-restore-10.2  6s
+pg-restore-10.6  6s
+pg-restore-11.1  6s
+pg-restore-11.2  6s
+pvc-backup       6h55m
+pvc-restore      6h55m
+update-status    6h55m
+```
+
+Also, verify that the necessary `Task` has been created,
+
+```console
+$ kubectl get task
+NAME          AGE
+$ kubectl get task.stash.appscode.com
+NAME             AGE
+pg-backup-9.6    10s
+pg-backup-10.2   10s
+pg-backup-10.6   10s
+pg-backup-11.1   10s
+pg-backup-11.2   10s
+pg-restore-9.6   10s
+pg-restore-10.2  10s
+pg-restore-10.6  10s
+pg-restore-11.1  10s
+pg-restore-11.2  10s
+pvc-backup       4d7h
+pvc-restore      4d7h
+```
+
+## Prepare Template
+
+We are going to use [GCS Backend](/docs/guides/latest/backends/gcs.md) to store the backed up data. You can use any supported backend you prefer. You just have to configure Storage Secret and `spec.backend` section of `BackupConfigurationTemplate` to match with your backend. To know which backed is supported by Stash and how to configure them, please visit [here](/docs/guides/latest/backends/overview.md).
+
+**Create Storage Secret:**
+
+At first, let's create a Storage Secret for the GCS backend,
+
+```console
+$ echo -n 'changeit' > RESTIC_PASSWORD
+$ echo -n '<your-project-id>' > GOOGLE_PROJECT_ID
+$ mv downloaded-sa-json.key > GOOGLE_SERVICE_ACCOUNT_JSON_KEY
+$ kubectl create secret generic -n demo gcs-secret \
+    --from-file=./RESTIC_PASSWORD \
+    --from-file=./GOOGLE_PROJECT_ID \
+    --from-file=./GOOGLE_SERVICE_ACCOUNT_JSON_KEY
+secret/gcs-secret created
+```
+
+**Create BackupConfigurationTemplate:**
+
+Now, we have to create a `BackupConfigurationTemplate` crd with a template for `Repository` and `BackupConfiguration` object.
+
+Below is the YAML of the `BackupConfigurationTemplate` object that we are going to create,
+
+```yaml
+apiVersion: stash.appscode.com/v1beta1
+kind: BackupConfigurationTemplate
+metadata:
+  name: pg-backup-template
+spec:
+  # ============== Template for Repository ==========================
+  backend:
+    gcs:
+      bucket: appscode-qa
+      prefix: stash-backup/${TARGET_NAMESPACE}/${TARGET_KIND}/${TARGET_NAME}
+    storageSecretName: gcs-secret
+  # ============== Template for BackupConfiguration =================
+  task:
+    name: pg-backup-11.2
```

Comment:
I think that task field should be templatized. I think we should add a `Version` field in AppBinding. https://github.com/kmodules/custom-resources/blob/master/apis/appcatalog/v1alpha1/appbinding_types.go#L28

Then we will be able to use the same BCT for any version of Postgres database.

Reply:
That make sense.

Reply:
Added Version field. https://github.com/kmodules/custom-resources/pull/21